site2/docs/tiered-storage-aliyun.md

```diff
@@ -0,0 +1,242 @@
+---
+id: tiered-storage-aliyun
+title: Use Aliyun OSS offloader with Pulsar
+sidebar_label: Aliyun OSS offloader
+---
+
+This chapter guides you through every step of installing and configuring the Aliyun Object Storage Service (OSS) offloader and using it with Pulsar.
+
+## Installation
+
+Follow the steps below to install the Aliyun OSS offloader.
+
+### Prerequisite
+
+- Pulsar: 2.8.0 or later versions
```

Comment:
```suggestion
- Pulsar: 2.8.0 or higher versions
```

Reply:
@Huanli-Meng Various writing manuals (including Google developer documentation style, IBM Style Guide, Apple Style Guide, The Chicago Manual of Style, etc) suggest using <"earlier" or "later"> rather than <"lower" or "higher"> for versions.

Reply:
I think use "later" is better